lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -123,24 +75,38 @@ namespace Js
         library->arrayIteratorPrototypeBuiltinNextFunction = VarTo<JavascriptFunction>(JavascriptOperators::GetProperty(library->arrayIteratorPrototype, PropertyIds::next, scriptContext));
     }
 
-    void JsBuiltInEngineInterfaceExtensionObject::InjectJsBuiltInLibraryCode(ScriptContext * scriptContext)
+    void JsBuiltInEngineInterfaceExtensionObject::InjectJsBuiltInLibraryCode(ScriptContext * scriptContext, JsBuiltInFile file)
     {
         JavascriptExceptionObject *pExceptionObject = nullptr;
-        if (jsBuiltInByteCode != nullptr)
+        FunctionBody* jsBuiltInByteCode = nullptr;
+
+        switch (file)
         {
-            return;
+            #define jsBuiltInByteCodeCase(class, type, obj) \
+            case (JsBuiltInFile::class##_##type): \
+            { \
+                if (jsBuiltIn##class##_##type##Bytecode != nullptr) \
+                    return; \
+                EnsureSourceInfo(); \
+                uint32 flags = fscrJsBuiltIn | (CONFIG_FLAG(CreateFunctionProxy) && !scriptContext->IsProfiling() ? fscrAllowFunctionProxy : 0); \
+                SRCINFO* hsi = sourceInfo; \
+                Js::ByteCodeSerializer::DeserializeFromBuffer(scriptContext, flags, (LPCUTF8)nullptr, hsi, (byte*)js::Library_Bytecode_##class##_##type, nullptr, &jsBuiltIn##class##_##type##Bytecode); \
+                jsBuiltInByteCode = jsBuiltIn##class##_##type##Bytecode; \
```

Comment:
Reading in the bytecode is handled here - as above this switch will automatically grow if new js files are added.